src/reactions/MessageReactionList.js

```diff
@@ -0,0 +1,101 @@
+/* @flow strict-local */
+import React, { PureComponent } from 'react';
+import { View } from 'react-native';
+import { TabNavigator, TabBarTop } from 'react-navigation';
+import type { NavigationScreenProp } from 'react-navigation';
+
+import ReactionUserList from './ReactionUserList';
+import { connect } from '../react-redux';
+import type { AggregatedReaction, Dispatch, GlobalState, Message, UserOrBot } from '../types';
+import { Screen, RawLabel } from '../common';
+import { getOwnUser } from '../selectors';
+import aggregateReactions from './aggregateReactions';
+import styles from '../styles';
+import { getAllUsersById } from '../users/userSelectors';
+import tabsOptions from '../styles/tabs';
+import Emoji from '../emoji/Emoji';
+import { objectFromEntries } from '../jsBackport';
+
+const getReactionsTabs = (
+  aggregatedReactions: $ReadOnlyArray<AggregatedReaction>,
+  targetReactionName: string,
+  usersById: Map<number, UserOrBot>,
+) => {
+  const reactionsTabs = objectFromEntries(
+    aggregatedReactions.map(aggregatedReaction => [
+      aggregatedReaction.name,
+      {
+        screen: () => (
+          <ReactionUserList reactedUserIds={aggregatedReaction.users} users={usersById} />
+        ),
+        navigationOptions: {
+          tabBarLabel: () => (
+            <View style={styles.row}>
+              <Emoji name={aggregatedReaction.name} />
+              <RawLabel text={`  ${aggregatedReaction.count}`} />
```

Comment:
These space characters look awfully suspicious as a way of trying to control this layout. :wink: Can we use, say, a style with padding instead?